package.json

```diff
@@ -30,7 +30,8 @@
     "@storybook/addon-knobs": "^5.1.7",
     "classnames": "^2.2.6",
     "enzyme-to-json": "^3.3.5",
-    "prop-types": "^15.7.2"
+    "prop-types": "^15.7.2",
+    "react-decorate": "^5.1.1"
```

Comment:
Did you implement the decorator with TS?

Reply:
I want to enrich some components with more features, after searching I found that using a HOC is the best way and the most optimal, the decorator makes the components render slowly

Reply:
Cool make sense